Update create-dns-records-at-any-dns-hosting-provider.md (#9616)
MicrosoftDocs/microsoft-365-docs: microsoft-365/admin/get-help-with-domains/create-dns-records-at-any-dns-hosting-provider.md

If you don't add a domain, people in your organization will use the onmicrosoft.com domain for their email addresses until you do. It's important to add your domain before you add users, so you don't have to set them up twice.

> If you have created users before and would like to change their domain, folow steps mentioned at [Change your email address to use your custom domain using the Microsoft 365 admin center](/admin/email/change-email-address?view=o365-worldwide#change-your-email-address-to-use-your-custom-domain-using-the-microsoft-365-admin-center)

Reviewer: ```suggestion
If you have previously created users and would like to change their domain, follow the steps described in [Change your email address to use your custom domain using the Microsoft 365 admin center](/microsoft-365/admin/email/change-email-address#change-your-email-address-to-use-your-custom-domain-using-the-microsoft-365-admin-center).
```
